.github/workflows/main.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [main]
+    branches: true
```

Comment:
i think this is unintended?

It might be my rebase, as I was testing adding protection rules to this branch before we merged the first few PRs.